WebRoot/css/zm.css

```diff
@@ -5033,6 +5033,20 @@ If you feel compelled to have a background color, please do so on a per skin lev
 	margin:26px 24px;
 }
 
+.ZmSharedFileRemoved {
+	text-decoration: line-through;
+}
+
+.ZmSharedFilePreview {
+	color: blue;
+	background: none;
```

Comment:
why are we using none values? are we trying to override some other styles?

Reply:
removed it.

Reply:
There are still more properties present in same class with same behaviour

Reply:
removed such properties.